pkg/core/mempool/mem_pool.go

```diff
@@ -217,6 +246,21 @@ func (mp *Pool) Remove(hash util.Uint256) {
 		senderFee := mp.fees[tx.Sender()]
 		senderFee.feeSum.Sub(senderFee.feeSum, big.NewInt(tx.SystemFee+tx.NetworkFee))
 		mp.fees[tx.Sender()] = senderFee
+		if feer.P2PSigExtensionsEnabled() {
+			// remove all conflicting hashes from mp.conflicts list
+			for _, attr := range tx.Attributes {
```

Comment:
I think, it is time to have `tx.GetAttribute` (may be replace `HasAttribute`).

Reply:
Added `tx.GetAttributes`.